ocode_python/mcp/manager.py

```
@@ -201,22 +211,35 @@ async def stop_server(self, name: str) -> MCPServerInfo:
-                info.status = "stopped"
-                info.pid = None
+                    info.status = "stopped"
+                    info.pid = None
+
+                except (psutil.NoSuchProcess, psutil.AccessDenied):
+                    info.status = "stopped"
+                    info.pid = None
+            else:
+                # Fallback: try OS kill command
+                try:
+                    import os
```

Pylint (3.3.7)

[warning] Redefining name 'os' from outer scope (line 9)

(W0621)